tv_apps/api-demos-tv/manifest.webapp

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "API Demos",
+  "description": "Demo app to show all APIs in TV",
+  "launch_path": "/index.html",
+  "type": "certified",
+  "developer": {
+    "name": "The Gaia Team",
+    "url": "https://github.com/mozilla-b2g/gaia"
+  },
+  "permissions": {
+    "desktop-notification":{}
+  },
+  "locales": {
+    "en-US": {
+      "name": "API Demos",
+      "description": "Demo app to show all APIs in TV"
+    }
+  },
+  "default_locale": "en-US",
+  "icons": {
+    "354": "/style/icons/developer.png"
+  }
+}
```

Comment:
Does not app start.

[1].case of "use notification web api":
- I think the following in manifest.webapp is missing.
  "messages": [
    { "notification": "/index.html" }
  ],
- This reason is due to the following code.
 gecko/b2g/components/AlertsHelper.jsm:L197
      listener.target = getNotificationURLFor(manifest.messages);

 gecko/b2g/components/AlertsHelper.jsm:L155
          gSystemMessenger.sendMessage(kNotificationSystemMessageName, {
              clicked: (detail.type === kDesktopNotificationClick),
              title: listener.title,
              body: listener.text,
              imageURL: listener.imageURL,
              lang: listener.lang,
              dir: listener.dir,
              id: listener.id,
              tag: listener.tag,
              timestamp: listener.timestamp,
              data: listener.dataObj
            },
            Services.io.newURI(listener.target, null, null),   //L155
            Services.io.newURI(listener.manifestURL, null, null)
          );


[2].case of "use IAC":
- I think something of "connect" in manifest.webapp is missing.
- Would not be a problem the following?
　 In the event handler for app.launch(), "notification" exists, but "connect" does not exist.